### ISSUE TITLE
Event Streaming or Non-interactive Data Collection

### ISSUE THREAD

**Issue by weslambert** [redcanaryco/mac-monitor#19]
**At a high level -- can you summarize your request?**
I realize the primary purpose of the project is to provide a means to analyze, correlate, and debug efficiently through the GUI, however, has it been considered to add the option to write to a log by default when a trace is running, or non-interactively stream events to an external destination so that further analysis can be performed with the data using another platform? I know it's not intended not to simply be a `tail` command for ESF, but it might be useful if the log may contain some correlational data.

**What is the current alternative solution?**
Perform ad-hoc exporting of telemetry, then copy to another platform for ingestion and processing.

**Comment by Brandon7CC** [comment]
Hi @weslambert! At that point it would technically be an EDR sensor without the ability to stream telemetry. This is unfortunately not a capability we're looking to expose in this tool alone. We'd be happy to consider any feature requests not directly related to enabling an EDR sensor though!

I'd suggest looking to Apple's ESLogger utility `/usr/bin/eslogger` or may of the tools from [Patrick Wardle / Objective-See](https://objective-see.org/products/utilities.html)!

**Comment by weslambert** [comment]
I am familiar with those, but figured I'd ask 😉. Thanks!

**Comment by Brandon7CC** [comment]
Of course! Feel free to shoot over any more if you think of anything

**Comment by mgraeber-rc** [comment]
Hey, @weslambert! I just wanted to give a shoutout to @Brandon7CC for releasing [AtomicESClient](https://github.com/redcanaryco/mac-monitor/tree/main/AtomicESClient) which is now included in this repo which can be used to get anyone wanting to learn Endpoint Security and Swift up and running with CLI-based ES event streaming. 🔥

**Comment by weslambert** [comment]
Awesome, thanks @mgraeber-rc and @Brandon7CC! I'll definitely check it out!
